README.md

```diff
@@ -23,85 +175,121 @@ Copy and paste into your Terraform configuration, insert the variables, and run
 module "graphdb" {
   source  = "Ontotext-AD/graphdb/aws"
   version = "0.0.1"
+
+  resource_name_prefix     = "graphdb"
+  aws_region               = "us-east-1"
+  ec2_instance_type        = "c5a.2xlarge"
+  graphdb_license_path     = "path-to-graphdb-license"
+  allowed_inbound_cidrs_lb = ["0.0.0.0/0"]
 }
 ```
 
-## Configuration
-
-This README provides more detailed information about the important variables and how to configure them for the Terraform AWS GraphDB module. Please ensure that you adapt the configuration to match your specific use case and requirements.
+Initialize the module and it's required providers with:
 
-### Important Variables (Inputs)
+`terraform init`
 
-The following are the important variables you should configure when using this module:
+Before deploying, make sure to inspect the plan output with:
 
-- `aws_region`: The region in which GraphDB will be deployed.
+`terraform plan`
 
-- `vpc_id`: The ID of the Virtual Private Cloud (VPC) where you want to create the AWS GraphDB cluster. This is a required variable.
-
-- `instance_type`: The instance type for the GDB cluster nodes. This should match your performance and cost requirements.
-
-- `node_count`: The number of instances in the cluster. Recommended is 3, 5 or 7 in order to have consensus according to the [Raft algorithm](https://raft.github.io/).
+After a careful review of the output plan, deploy with:
 
+`terraform apply`
 
+Once deployed, you should be able to access the environment at the generated FQDN that has been outputted at the end.
 
 ## Examples
 
-This repository includes usage examples in the `examples/` directory, such as:
-
-- [VPC with Multiple Availability Zones](./examples/vpc-with-multiple-az): Demonstrates creating a VPC with multiple Availability Zones and provisioning an AWS GraphDB cluster.
-
 Here's an example of how to configure the module in your Terraform project:
 
 #### Example Configuration
 
 ##### terraform.tfvars
+
 ```hcl
 
 aws_region = "us-east-1"
 
-azs = ["us-east-1a", "us-east-1b", "us-east-1c"]
-
 resource_name_prefix = "my-prefix"
 
 graphdb_license_path = "/path/to/your/license.license"
 
-tags = {
-  Name = "MyInstance"
-  Environment = "Production"
-}
-
 ami_id = "GraphDB-AMI"
+
+ec2_instance_type = "c5a.2xlarge"
+
+allowed_inbound_cidrs_lb = ["0.0.0.0/0"]
+
 ```
 
-Ensure that you adjust the values of the variables to match your specific requirements.
+## Configuration
+
+This README provides more detailed information about the important variables and how to configure them for the Terraform
+AWS GraphDB module. Please ensure that you adapt the configuration to match your specific use case and requirements.
+
+## GraphDB Configurations
+
+`graphdb_properties_path = "<path_to_custom_graphdb_properties_file>"`
+
+Or by setting the GDB_JAVA_OPTS environment variable with graphdb_java_options. For example, if you want to print the
+command line flags, use:
+
+`graphdb_java_options = "-XX:+PrintCommandLineFlags"`
```

Comment:
You should mention those will be appended to the ones set from the user data script.